bot/exts/backend/sync/_cog.py

```diff
@@ -20,33 +21,39 @@ class Sync(Cog):
 
     def __init__(self, bot: Bot) -> None:
         self.bot = bot
+        self.guild: Guild = None
```

Comment:
Shouldn't this be `Guild | None`?

Reply:
Yea, fixed in https://github.com/python-discord/bot/commit/094e76e432054d713859f15883da63e70f59f76f